### PR TITLE
Add components, actions and selectors for controlled MapTimeline

### PR DESCRIPTION
**Pull request by vdubr**

### CODE REVIEW

File: src/components/timeline/maptimeline/LayerRowPeriodItem.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+		standardized = ownProps.itemPeriod;
+	}
+	const active = mapTimelineSelectors.getTimelineLayerPeriodActive(
+		ownProps.mapKey,
+		ownProps.layer,
+		ownProps.itemPeriod
+	);
+
+	return {
+		standardized,
```
Comment (vlach1989) [inline src/components/timeline/maptimeline/LayerRowPeriodItem.js:27]: what is standardized?

File: src/components/timeline/maptimeline/LayerRowPeriodItem.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+		active,
+	};
+};
+
+const mapDispatchToProps = (dispatch, ownProps) => {
+	return {
+		onMount: () => {
+			//ensure layers
+		},
+		ensurePeriod: periodKey => {
```
Comment (vlach1989) [inline src/components/timeline/maptimeline/LayerRowPeriodItem.js:37]: why is the method called ensure when dispatching use?

**Review by vlach1989 — CHANGES_REQUESTED** [review]

File: src/components/timeline/maptimeline/LayerRowPeriodItem.js (hunk 1 above)
Comment (vdubr) [reply src/components/timeline/maptimeline/LayerRowPeriodItem.js]: Renamed to the parsedPeriod 1152a2c1acdb5bc5991bfe3ebea8e5eb9f20a4ad

File: src/components/timeline/maptimeline/LayerRowPeriodItem.js (hunk 2 above)
Comment (vdubr) [reply src/components/timeline/maptimeline/LayerRowPeriodItem.js]: Renamed to usePeriod  : 1152a2c1acdb5bc5991bfe3ebea8e5eb9f20a4ad

**Review by vlach1989 — APPROVED** [review]

**Pull request merged**